test/e2e/toolchain_test.go

```diff
@@ -45,10 +47,7 @@ func TestToolchain(t *testing.T) {
 		// then
```

Comment:
Minor: `then` is mis-placed

Reply:
not sure how is it misplaced?

Reply:
sorry .. I got it wrong .. it's fine :+1: